src/FS.Embedded/build/netstandard2.0/Microsoft.Extensions.FileSystem.Embedded.targets

```diff
@@ -0,0 +1,61 @@
+<Project>
+
+  <UsingTask
+    AssemblyFile="Microsoft.Extensions.FileSystem.Embedded.Manifest.Task"
+    TaskName="Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.GenerateEmbeddedResourcesManifest" />
+
+  <PropertyGroup>
+    <GenerateEmbeddedFilesManifest Condition="'$(GenerateEmbeddedFilesManifest)' == ''">false</GenerateEmbeddedFilesManifest>
+    <EmbeddedFilesManifestFileName Condition="'$(EmbeddedFilesManifestFileName)' == ''">Microsoft.Extensions.FileSystem.Embedded.Manifest.xml</EmbeddedFilesManifestFileName>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(GenerateEmbeddedFilesManifest)' == 'true'">
+    <EmbeddedResource
+      Include="$(IntermediateOutputPath)$(EmbeddedFilesManifestFileName)"
+      IsManifest="true"
+      Condition="@(EmbeddedResource->WithMetadataValue('IsManifest','true')->Count()) == 0"
+      LogicalName="$(EmbeddedFilesManifestFileName)" />
+  </ItemGroup>
+
+  <Target Name="CreateGeneratedManifestInfoInputsCacheFile">
+    <PropertyGroup>
+      <GeneratedManifestInfoInputsCacheFile>$(IntermediateOutputPath)$(MSBuildProjectName).EmbeddedFilesManifest.cache</GeneratedManifestInfoInputsCacheFile>    
+  </PropertyGroup>    
+    
+    <Hash ItemsToHash="@(EmbeddedResource)">
+      <Output TaskParameter="HashResult" PropertyName="_EmbeddedGeneratedManifestHash" />
+    </Hash>
+
+    <WriteLinesToFile Lines="$(_EmbeddedGeneratedManifestHash)" File="$(GeneratedManifestInfoInputsCacheFile)" Overwrite="True" WriteOnlyWhenDifferent="True" />
+
+    <ItemGroup>
+      <FileWrites Include="$(GeneratedManifestInfoInputsCacheFile)" />
+    </ItemGroup>
+  </Target>
+
+  <Target 
+    Name="_GenerateEmbeddedFilesManifest"
+    DependsOnTargets="CreateGeneratedManifestInfoInputsCacheFile"
+    AfterTargets="PrepareResourceNames"
+    Condition="'$(GenerateEmbeddedFilesManifest)' == 'true'"
+    Inputs="$(GeneratedManifestInfoInputsCacheFile)"
+    Outputs="@(EmbeddedResource->WithMetadataValue('IsManifest', 'true'))">
+    <ItemGroup>
+      <__FilesForManifest Include="@(EmbeddedResource)" />
+      <__FilesForManifest Remove="@(EmbeddedResource->WithMetadataValue('ExcludeFromManifest','true'))" />
+      <__FilesForManifest Remove="@(EmbeddedResource->WithMetadataValue('IsManifest','true'))" />
+      <__ManifestFiles Include="@(EmbeddedResource->WithMetadataValue('IsManifest', 'true'))" />
+    </ItemGroup>
+
+    <Error Condition="'@(__ManifestFiles->Count())' > 1" Text="Multiple manifest files found. Only one manifest file per assembly is allowed: @(__ManifestFiles->TargetPath)" />
```

Comment:
`@(__ManifestFiles->TargetPath)` is this valid MSBuild syntax? I don't recognized it.